src/node_buffer.cc

```diff
@@ -741,6 +749,46 @@ void SlowByteLengthUtf8(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(args[0].As<String>()->Utf8Length(env->isolate()));
 }
 
+#if NODE_HAS_SIMD_NEON
+uint32_t FastByteLengthUtf8(Local<Value> receiver,
+                            const v8::FastOneByteString& source) {
+  uint8_t* data =
+      const_cast<uint8_t*>(reinterpret_cast<const uint8_t*>(source.data));
```

Comment:
`vld1q_u8` is documented to take a `const uint8_t*` argument. What am I missing?

Reply:
Updated the line. My bad!